deploy/terraform/terraform-units/modules/sap_deployer/variables_local.tf

```
@@ -15,6 +15,9 @@ locals {
   // Default option(s):
   enable_secure_transfer    = try(var.options.enable_secure_transfer, true)
   enable_deployer_public_ip = try(var.options.enable_deployer_public_ip, false)
+  deployer_public_ip_address = try(var.options.enable_deployer_public_ip, "")
+  Agent_IP = try(var.options.Agent_IP, "")
```

This should be like this we set that using export TF_VAR_Agent_IP

```suggestion
  Agent_IP = try(var.Agent_IP, "")
```